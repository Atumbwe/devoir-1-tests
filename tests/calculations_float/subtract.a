6546.3 534.654 -