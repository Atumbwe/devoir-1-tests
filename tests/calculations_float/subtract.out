 -------------------------
 <    6011.646000000001    >
 -------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||